data/json/flags.json

```
@@ -7,6 +7,7 @@
   },
   {
     "id": "EFFECT_INVISIBLE",
+    "context": [ ],
```

Linter expects two whitespaces:
```suggestion
    "context": [  ],
```